templates/cassiopeia/css/template.css

```diff
@@ -83,7 +83,7 @@ html {
   -webkit-text-size-adjust: 100%;
   -ms-text-size-adjust: 100%;
   -ms-overflow-style: scrollbar;
-  -webkit-tap-highlight-color: transparent; }
```

Comment:
Why do we need to make a change in the frontend template?

Reply:
I think it was changed during <code>npm install </code>.

Reply:
Better to revert it then.